src/types.ts

```diff
@@ -30,6 +34,7 @@ export interface EventLog {
   blockNumber: bigint;
   eventName: string;
   args: Record<string, any>;
+  transactionHash: string;
```

Comment:
Why do we need this `transactionHash`?

Reply:
it might be easier in the future to fetch transaction data or redirect user to etherscan using transactionHash.